src/components/calcite-action/calcite-action.tsx

```diff
@@ -45,11 +45,6 @@ export class CalciteAction {
    */
   @Prop({ reflect: true }) indicator = false;
 
-  /**
-   * Label of the action, exposed on hover.
-   */
-  @Prop() label: string;
```

Comment:
@pr3tori4n maybe we should keep this. It gives us more flexibility to set more of a hover description or some additional information to just the text being displayed. @asangma what do you think?